electron/main.ts

```diff
@@ -355,14 +357,14 @@ ipcMain.handle('install', async (event, args) => {
   const { defaultInstallPath, maxWorkers } = await getSettings('default')
   const workers = maxWorkers === 0 ? '' : `--max-workers ${maxWorkers}`
 
-  const logPath = `${heroicGamesConfigPath}${game}.log`
+  const logPath = `"${heroicGamesConfigPath}${game}.log"`
   let command = `${legendaryBin} install ${game} --base-path '${path}' ${workers} -y &> ${logPath}`
   if (path === 'default') {
     command = `${legendaryBin} install ${game} --base-path ${defaultInstallPath} ${workers} -y |& tee ${logPath}`
   }
   console.log(`Installing ${game} with:`, command)
   try {
-    await execAsync(command, { shell: '/bin/bash' })
+    await execAsync(command, { shell: shell })
```

Comment:
The only thing we need to check if this breaks HEroic on Ubuntu, because it requires that everything runs on bash. So, if this breaks, we will need create a global variable like: `const isMac = os.platform() === 'darwin'` and use it to choose the shell.

Reply:
Heroic on linux continues to use bash, it only changes to zsh if running on mac as you said.
Check 'shell' variable at electron/utils.ts